src/Chapter3.hs

```diff
@@ -1098,19 +1111,17 @@ data WeekDay
   deriving (Eq, Ord, Enum, Bounded, Show)
 
 isWeekend :: WeekDay -> Bool
-isWeekend weekDay
-  | weekDay > Friday = True
-  | otherwise        = False
+isWeekend weekDay = weekDay > Friday
 
 nextDay :: WeekDay -> WeekDay
-nextDay = succ
+nextDay Sunday = minBound
```

Comment:
```suggestion
nextDay day
    | day == maxBound = minBound
    | otherwise = succ day
```

This is also a bit more maintainable. If you decide, for example, to have the "American" system of week days in the data type constructors, you won't need to change this function at all! 🙂